File: SoloLearn Codes/No Numerals/NoNumerals.java
import java.util.Arrays;
import java.util.List;
import java.util.Scanner;
import java.util.stream.Collectors;

public class NoNumerals {

    public static void main(String[] args) {
        List<String> numbers = List.of("zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten");
        System.out.println(Arrays.stream(new Scanner(System.in)
                .nextLine().split(" "))
                .map(t -> t.matches("[0-9]+") ? numbers.get(Integer.parseInt(t)) : t)
                .collect(Collectors.joining(" ")));
    }
}

File: SoloLearn Codes/No Numerals/no_numerals.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <ctype.h>

int main() {
    char arr[11][8] = { "zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten" };
    char s[128]; 
    fgets(s, 128, stdin);
    char *token = strtok(s, " ");
    while (token != NULL) {
        if (isdigit(token[0])) printf("%s ", arr[atoi(token)]);
        else printf("%s ", token);
        token = strtok(NULL, " ");
    }
    return 0;
}